src/main/java/edu/kpi/testcourse/rest/AuthenticatedApiController.java

```diff
@@ -71,4 +77,21 @@ public AuthenticatedApiController(
       );
     }
   }
+
+  @Get(value = "/urls", processes = MediaType.APPLICATION_JSON)
+  public HttpResponse<String> getAll(
+    @Body UrlShortenRequest request,
+    Principal principal,
+    HttpRequest<?> httpRequest,
+    Logic logic) throws UrlRepository.PermissionDenied {
+    try {
+      var aliasList = logic.getAllAliasesForUser(principal.getName());
+      return HttpResponse.created(json.toJson(aliasList));
```

Comment:
According to the [spec](https://github.com/future-stardust/url-shortener-specification/blob/main/openapi.yaml), we should return the 200 HTTP response.
```suggestion
      return HttpResponse.ok(json.toJson(aliasList));
```